sonaranalyzer-dotnet/src/SonarAnalyzer.Utilities/Rules.Description/S2255_cs.html

```diff
@@ -19,8 +19,7 @@ <h2>Recommended Secure Coding Practices</h2>
 <p>Using cookies only for session IDs doesn't make them secure. Follow <a
 href="https://www.owasp.org/index.php/Session_Management_Cheat_Sheet#Cookies">OWASP best practices</a> when you configure your cookies.</p>
 <p>As a side note, every information read from a cookie should be <a
-href="https://www.owasp.org/index.php/Input_Validation_Cheat_Sheet">Sanitized</a>. See injection rules such as {rule:csharpsquid:S3649} for more
-information.</p>
+href="https://www.owasp.org/index.php/Input_Validation_Cheat_Sheet">Sanitized</a>.</p>
```

Comment:
VB file for this rule got updated in commit `Update S2255 - VB net does not have injection rules`